CONTRIBUTING.md

```diff
@@ -32,7 +32,7 @@ The committer role enables the participant to commit code directly to the reposi
 * Monitor Slack (delayed response is perfectly acceptable).
 * Triage GitHub issues and perform pull request reviews for other committers and the community.
 * Make sure that ongoing PRs are moving forward at the right pace or close them.
-* In general continue to be willing to spend at least 25% of one's time working on the project (~1.25 business days per week).
+* In general continue to be an active contributor to the project in general and the code base in particular.
```

Comment:
In general ... in general ?

Reply:
good catch - let's change it to:

"* Remain an active contributor to the project in general and the code base in particular."

Reply:
But.... Then we lose the wonderful passive voice?  :>

I've made that change.